CMakeLists.txt

```diff
@@ -219,11 +224,15 @@ include(cmake/external_gtest.cmake)
 include(cmake/external_json.cmake)
 include(cmake/external_eigen.cmake)
 include(cmake/external_mkldnn.cmake)
-if (NGRAPH_USE_PREBUILT_LLVM OR DEFINED LLVM_TARBALL_URL)
-    include(cmake/external_llvm_prebuilt.cmake)
-else()
-    include(cmake/external_llvm.cmake)
+
+if (NOT NGRAPH_DEX_ONLY)
```

Comment:
Always include LLVM. If the CPU backend does not depend on it then it won't be built.

Reply:
If LLVM is always included - then cmake will need to check if it needs to update and then download if needed even if it's not used by the backend - no?

Reply:
It will only try to download if something depends on it, so for DEX_ONLY it will not download